packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,70 @@
+# This file contains the analysis options used by the Dart analyzer
+# to decide what errors, warnings, and hints should be shown.
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: false
+  errors:
+    # we allow overriding fields (if they use super, ideally...)
+    strong_mode_invalid_field_override: ignore
+    # we allow type narrowing
+    strong_mode_invalid_method_override: ignore
+    strong_mode_static_type_error: ignore
+    strong_mode_down_cast_composite: ignore
+    # we allow having TODOs in the code
+    todo: ignore
```

Comment:
My impression from various conversations was that turning this on caused the `flutter analyze` output to be very noisy for both us and our customers. It caused legitimate errors and warnings to be missed and worse caused devs to stop using `flutter analyze`.

`flutter analyze` currently ignores todo comments and I recommend that we continue to do so.